third_party/chromium/media/filters/source_buffer_stream.h

```diff
@@ -188,6 +188,12 @@ class MEDIA_EXPORT SourceBufferStream {
     memory_limit_ = memory_limit;
   }
 
+#if defined(STARBOARD)
+  size_t GetMemoryLimit() {
+    return memory_limit_;
```

Comment:
In source_buffer_stream.cc, `memory_limit_` can be overwritten in `SourceBufferStream::UpdateVideoConfig()`, and the change won't be reflected here.

Consider not overwriting memory limit if the memory limit is explicitly set.

Reply:
done.